src/R/Components/Impl/PackageManager/Implementation/View/PackageList.xaml.cs

```diff
@@ -88,5 +91,17 @@ public partial class PackageList : UserControl {
             var package = GetPackage(e);
             Model?.UnloadAsync(package).DoNotWait();
         }
+
+        protected override AutomationPeer OnCreateAutomationPeer() {
```

Comment:
To fire property change event so narrator re-reads the state indicating that the package is now loaded or installed.